Week3/homework/step3.js

```diff
@@ -1,12 +1,13 @@
 'use strict';
 
 function createBase(base) {
-  // Replace this comment and the next line with your code
-  console.log(base);
+  function addNumToBase(num) {
```

Comment:
Nice work. You could even `return` an anonymous function here, instead of naming it and returning it later.